ci-matrix-config.yml

```diff
@@ -0,0 +1,17 @@
+include:
+  - fqbn: esp32:esp32:featheresp32
+
+exclude:
+  # Not enough memory available on these boards
+  - sketch: examples/read_3_different_iic_sensors/read_3_different_iic_sensors.ino
```

Comment:
This could be (if no bug 🐛 ) be also a list:
```
- sketch: 
   - examples/read_3_different_iic_sensors/read_3_different_iic_sensors.ino
   - examples/read_iic_sensor_c_style/read_iic_sensor_c_style.in
```
And you can save another entry for the second example, and the same set of boards.